scene/gui/text_edit.cpp

```diff
@@ -1173,12 +1169,7 @@ void TextEdit::_notification(int p_what) {
 						}
 					}
 
-					// Carets
```

Comment:
That comment should be kept, it's for the whole chunk of code that handles carets.

Reply:
Oh, I probably removed it by accident. Will restore.